pkg/services/ngalert/api/api_ruler.go

```diff
@@ -43,8 +43,12 @@ func (srv RulerSrv) RouteDeleteRuleGroupConfig(c *models.ReqContext) response.Re
 	}
 	ruleGroup := c.Params(":Groupname")
 	if err := srv.store.DeleteRuleGroupAlertRules(c.SignedInUser.OrgId, namespace.Uid, ruleGroup); err != nil {
-		return response.Error(http.StatusInternalServerError, "failed to delete group alert rules", err)
```

Comment:
The validation below was a result of this - previously this endpoint would return everything OK deleted when we deleted nothing.